src/Appwrite/Platform/Tasks/Install.php

```diff
@@ -157,12 +158,14 @@ public function action(string $httpPort, string $httpsPort, string $organization
                 }
 
                 if ($var['filter'] === 'token') {
-                    $input[$var['name']] = Auth::tokenGenerator();
+                    $token = new Token();
+                    $input[$var['name']] = $token->generate();
                     continue;
                 }
 
                 if ($var['filter'] === 'password') {
-                    $input[$var['name']] = Auth::passwordGenerator();
+                    $password = new Password();
+                    $input[$var['name']] = $password->generate();
```

Comment:
I said outside of the `foreach` because this is creating a new instance each time.